### ISSUE TITLE
Multiplayer Safe?

### ISSUE THREAD

**Issue by jkroepke** [afex/graftorio#3]
https://i.imgur.com/jVmPVvv.png

**Comment by afex** [comment]
can you provide the details provided in the log file that error mentions?

**Comment by jkroepke** [comment]
No, 10 months ago..

**Comment by msmollin** [comment]
Have there been changes since then that would improve this support? I'd be wiling to give it a shot on my multiplayer server (can snapshot the VM and roll back if it breaks it).

**Comment by afex** [comment]
i'm unable to confirm that this report is a bug.

@msmollin if you are willing to test on your server, i would appreciate it.

**Comment by lluchs** [comment]
I'm seeing the same issue. Here's what the log says:

```
  28.793 Error ClientMultiplayerManager.cpp:1097: mod-graftorio was registered for the following events when the map was saved but has not re-registered them as a result of loading: Custom event (ID 160)
  28.793 Error ClientMultiplayerManager.cpp:96: MultiplayerManager failed: "" + multiplayer.script-event-mismatch + "
" + "
mod-graftorio"
  28.795 Info ClientMultiplayerManager.cpp:572: UpdateTick(10337) changing state from(ConnectedLoadingMap) to(Failed)
```

**Comment by patschi** [comment]
I'm also experiencing same issues with latest 0.7.14 on a multiplayer server.

**Comment by afex** [comment]
fixed via #9
